src/content/api/module-methods.md

```diff
@@ -166,6 +166,8 @@ The goal of CommonJS is to specify an ecosystem for JavaScript outside the brows
 
 ### `require`
 
+<!-- eslint-skip -->
```

Comment:
Why is this needed?

Reply:
Because it is invalid javascript syntax:
```
require(dependency: String);
```

Reply:
changing ```js to ```ts|typescript works too or not supported?

Reply:
`ts` or `typescript` is another valid option in those cases.

And we do have two occurrences in this project:

![image](https://user-images.githubusercontent.com/1091472/92541494-c0a11180-f278-11ea-8c01-4cd348afefa7.png)

Do you prefer `ts`/`typescript`?

Reply:
since there's already `typescript` in the codebase, go ahead with that